xclim/core/utils.py

```diff
@@ -162,3 +164,208 @@ def sfcwind_2_uas_vas(wind: xr.DataArray = None, windfromdir: xr.DataArray = Non
     wind.attrs["units"] = "m s-1"
 
     return uas, vas
+
+
+@declare_units(
```

Comment:
This should go into indices, no ?

Reply:
Well. Maybe that's a discussion we could have. I feel this is the same kind of function as the surface wind ones or `xc.indices.tas`. They do not perform any indice-like stats and are completely dimension-independent, thus I think they don't really belong to the indices library.  Here I used the "declare_units" simply not to have to manage that part as it is important here, while not really in `tas()` and the surface wind utils.

Reply:
We could make a `xclim.indices.utils` file and put them there, but I would vote against throwing them in the same places as real indices.